src/pages/home/report/ReportActionItemMessageEdit.js

```diff
@@ -232,7 +232,7 @@ function ReportActionItemMessageEdit(props) {
 
         // When user tries to save the empty message, it will delete it. Prompt the user to confirm deleting.
         if (!trimmedNewDraft) {
-            ReportActionContextMenu.showDeleteModal(props.reportID, props.action, false, deleteDraft, () => InteractionManager.runAfterInteractions(() => textInputRef.current.focus()));
+            ReportActionContextMenu.showDeleteModal(props.reportID, props.action, false, deleteDraft);
```

Comment:
Why is this change required?

Reply:
In ReportActionItem, hideDeleteModal is called and then this cancel callback is called. focusing on non-existent element raises an error

Reply:
What does this have to do with the previous PR changes?

Reply:
This is a hidden bug

Reply:
Please do not make changes that does not reflect the PR scope unless we are fixing another bug - it seems we are removing a feature. We revert the first PR and find a solution to this. I think your solution won't work due to how useEffects propogate.

At first ReportActionContextMenu.isActiveReportAction will be true, but at the end in the useEffect it will be false. The main composer won't show.

Reply:
> At first ReportActionContextMenu.isActiveReportAction will be true, but at the end in the useEffect it will be false. The main composer won't show.

That's true. So I don't use `ReportActionContextMenu.isActiveReportAction` in useEffect

> Please do not make changes that does not reflect the PR scope unless we are fixing another bug

We can't solve this issue because deleting the draft this way raises an error in android native

Reply:
> We revert the first PR and find a solution to this

What's your solution?

Reply:
Please test your solution in android native

Reply:
The old PR was reverted by @marcochavezf

Reply:
We need to find a fix for the regression. Let's continue working on this on the issue